tiers/tests/test_tiers.py

```diff
@@ -15,29 +15,29 @@ class TiersTests(TiersTestCaseBase):
 
     def test_non_expired_tier(self):
         t = TierFactory()
-        assert t.tier_enforcement_exempt == False
-        assert t.has_tier_expired() == False
-        assert t.has_tier_grace_period_expired() == False
+        assert t.tier_enforcement_exempt is False
```

Comment:
`assert x is False` is not so Pythonic! However `assert not x` is.

```suggestion
        assert not t.tier_enforcement_exempt
```

Same for all the changes. If you're up to it.